lib/little-loader.js

```diff
@@ -51,27 +55,30 @@
    * @param {Object}    context   (Optional) Callback context
    * @returns {void}
    */
-  window._lload = function (src, callback, context) {
+  var _lload = function (src, callback, context) {
     /*eslint max-statements: [2, 20]*/
     context = context || this;
```

Comment:
It technically is mutating `arguments`, but since we don't reference `arguments` anywhere and no outside code could possibly have a handle on it (AFAIK), it doesn't matter.